frost-client/src/init.rs

```diff
@@ -0,0 +1,98 @@
+use std::error::Error;
+
+use eyre::eyre;
+
+use crate::{
+    args::Command,
+    config::{CommunicationKey, Config, Registry},
+};
+
+pub(crate) async fn init(args: &Command) -> Result<(), Box<dyn Error>> {
```

Comment:
The permissions for the config file should be `-rw-------` because it contains a private key

Reply:
Good point. I fixed this cb88a454 by "forking" `write_atomic` and making it use tempfile's `persist()` which does that by default.